modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -20,15 +20,17 @@ In {OCP} 4.13, the `stable` channel has been removed. Before upgrading to {OCP}
 ====
 
 |===
-|*OpenShift GitOps* 8+|*Component Versions*|*OpenShift Versions*
+|*OpenShift GitOps* 7+|*Component Versions*|*OpenShift Versions*
 
-|*Version* |*`kam`*    |*Helm*  |*Kustomize* |*Argo CD*|*Argo Rollouts*|*ApplicationSet* |*Dex*     |*RH SSO* |
+|*Version* |*`kam`*    |*Helm*  |*Kustomize* |*Argo CD*|*Argo Rollouts*|*Dex*     |*RH SSO* |
 
-|1.11.0 |0.0.51 TP |3.13.2 GA |5.2.1 GA |2.9.2 GA |1.6.0 TP |NA |2.36.0 GA |7.6.0 GA |4.12-4.14
+|1.12.0 |0.0.51 TP |3.14.0 GA |5.2.1 GA |2.10.3 GA |1.6.0 TP |2.36.0 GA |7.6.0 GA |4.12-4.14
 
-|1.10.0 |0.0.50 TP |3.12.1 GA |5.1.0 GA |2.8.3 GA |1.5.0 TP |NA |2.35.1 GA |7.5.1 GA |4.12-4.14
+|1.11.0 |0.0.51 TP |3.13.2 GA |5.2.1 GA |2.9.2 GA |1.6.0 TP |2.36.0 GA |7.6.0 GA |4.12-4.14
 
-|1.9.0    |0.0.49 TP |3.11.2 GA|5.0.1 GA   |2.7.2 GA |1.5.0 TP |NA     |2.35.1 GA |7.5.1 GA |4.12-4.14
+|1.10.0 |0.0.50 TP |3.12.1 GA |5.1.0 GA |2.8.3 GA |1.5.0 TP |2.35.1 GA |7.5.1 GA |4.12-4.14
+
+|1.9.0  |0.0.49 TP |3.11.2 GA |5.0.1 GA |2.7.2 GA |1.5.0 TP |2.35.1 GA |7.5.1 GA |4.12-4.14
```

Comment:
@harrietgrace @iam-veeramalla @reginapizza @varshab1210 Should we remove 1.9.0 from the table as we support only n, n-1, and n-2 versions?

Reply:
Yes, we would need to remove it
cc: @Dhruv-Soni11

Reply:
Fixed.